strings/tcg/set_two.json

```
@@ -297,7 +297,7 @@
 			"id": "hivemind",
 			"name": "Xenomorph Hivemind",
 			"desc": "Recently discovered to have sapience, this pulsating orb will dig into the earth and rapidly spread resin throughout planets.",
-			"rules": "Defender",
+			"rules": "{$Defender} Tap: Add 1 Xenomorph monster from your Deck to your hand.",
```

```suggestion
			"rules": "{$Defender} Tap: Add 1 Xenomorph creature from your Deck to your hand.",
```
Swap out monster since you're already using a keyword for abomination on other cards, this looks like another keyword instead of what I assume you're intending as a creature